test/test_main.py

```diff
@@ -1,7 +1,6 @@
 import pytest
 from itertools import zip_longest
 
-
```

Comment:
Seems this line is creating a conflict..

Reply:
Have updated my fork. Seems to have resolved the conflict.